style.css

```diff
@@ -0,0 +1,78 @@
+*{
+    margin: 0%;
+    padding: 0%;
+    box-sizing: border-box;
+}
+body{
+    display: flex;
+    justify-content: center;
+    align-self: center;
+    min-height: 100vh;
+
+    background: #222;
+}
+.container{
+    position: relative;
```

Comment:
Does this line of code has any effect

Reply:
NO,I removed this line but did'nt see any change...why so?